Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "coset"
-version = "0.3.4"
+version = "0.3.5"
```

Comment:
Please could you add a stub entry to `CHANGELOG.md` (with TBD as the date for 0.3.5)?

Reply:
Added an entry to CHANGELOG.md.